odk2odm/odk_api.py

```diff
@@ -42,12 +42,23 @@
 import sys, os
 import requests
 import json
+import zlib
+import qrcode
+
+import urllib
 
 def projects(base_url, aut):
     """Fetch a list of projects on an ODK Central server."""
     url = f'{base_url}/v1/projects'
     return requests.get(url, auth = aut)
 
+def project_id(base_url, aut, projectName):
+    """Fetch the id of a project based on the name on an ODK Central server."""
+    url = f'{base_url}/v1/projects'
+    projects = requests.get(url, auth = aut).json()
+    projectId = [p for p in projects if p['name']== projectName][0]['id']
+    return projectId
+
```

Comment:
What is this function for? I can't see any reason why you would want to filter out a project, and select its primary key using another field (fixed to 'name'). Normally you would simply query a project record based on its primary key. I am guessing you can do this with the following url construct:
url = f'{base_url}/v1/projects/{id}'